daemon/cmld.c

```diff
@@ -1120,6 +1148,14 @@ cmld_container_destroy_cb(container_t *container, container_callback_t *cb, UNUS
 	if (cb)
 		container_unregister_observer(container, cb);
 
+	if (container_get_token_is_init(container)) {
+		smartcard_scd_token_block_remove(cmld_smartcard, container);
+	}
+
+	if (container_get_token_uuid(container)) {
```

Comment:
move uuid cleanup to container_destroy()

Reply:
moved it to container_free(), as I believe that is the more appropriate solution